code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -186,67 +266,117 @@
 	desc = "An experimental suit of armor that violently throws back attackers."
 	reactivearmor_cooldown_duration = 5 SECONDS
 	var/repulse_force = MOVE_FORCE_EXTREMELY_STRONG
+	cooldown_message = "<span class='danger'>The repulse generator is still recharging! It fails to generate a strong enough wave!</span>"
+	emp_message = "<span class='warning'>The repulse generator is reset to default settings...</span>"
+
+/obj/item/clothing/suit/armor/reactive/repulse/reactive_activation(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", damage = 0, attack_type = MELEE_ATTACK)
+	playsound(get_turf(owner),'sound/magic/repulse.ogg', 100, 1)
+	owner.visible_message("<span class='danger'>[src] blocks [attack_text], converting the attack into a wave of force!</span>")
+	var/turf/T = get_turf(owner)
+	var/list/thrown_items = list()
+	for(var/atom/movable/A as mob|obj in orange(7, T))
```

Comment:
``/atom/movable`` is only mob and obj anyway, the ``as mob|obj`` is useless here.

Reply:
yeah idk why they put this here :shrugs: